docusaurus/docs/shared/create-plugin.md

```diff
@@ -0,0 +1,36 @@
+Tooling for modern web development can be tricky to wrap your head around. While you certainly can write your own webpack configuration, for this guide, you'll be using grafana create-plugin tool
+
+The Grafana [create-plugin tool](https://www.npmjs.com/package/@grafana/create-plugin) is a CLI application that simplifies Grafana plugin development, so that you can focus on code. The tool scaffolds a starter plugin and all the required configuration for you.
+
+1. In the plugin directory, create a plugin from template using create-plugin. When prompted for the kind of plugin, select `datasource`:
```

Comment:
🤦 this is also in the main docs on Grafana.com - but we tell people to create datasource even in the panel tutorial.

given all the shared components are only used in draft content. I'm going to assume that's where we fix things like this rather than block this PR